src/Card.js

```diff
@@ -0,0 +1,313 @@
+import Phaser from 'phaser'
+import { boardDrag, cardDimensions, hoverButtonRadius, cardBackFrame, cardDepth, activeDepth, hoverOffset, canvasHeight, inHandAdjustment, canvasWidth, inHandRange } from './Constants'
```

Comment:
Dont be afraid of breaking this onto multiple lines. In my experience taller code > wider code